servicetalk-concurrent-internal/src/main/java/io/servicetalk/concurrent/internal/ConcurrentUtils.java

```diff
@@ -89,13 +78,10 @@ private ConcurrentUtils() {
      * @param <T> Type of items stored in the {@link Queue}.
      * @param <R> Type of the object holding the {@link int} referred by {@link AtomicIntegerFieldUpdater}.
      * @return Number of items drained from the queue.
-     * @throws RuntimeException All {@link Throwable} thrown by {@code forEach} {@link Consumer} are added as suppressed
-     * causes.
      */
-    public static <T, R> long drainSingleConsumerQueueDelayThrow(final Queue<T> queue, final Consumer<T> forEach,
-                                                                 final AtomicIntegerFieldUpdater<R> drainActiveUpdater,
-                                                                 final R flagOwner) {
-        RuntimeException cause = null;
+    public static <T, R> long drainSingleConsumerQueue(final Queue<T> queue, final Consumer<T> forEach,
+                                                       final AtomicIntegerFieldUpdater<R> drainActiveUpdater,
+                                                       final R flagOwner) {
         long drainedCount = 0;
         do {
             if (!drainActiveUpdater.compareAndSet(flagOwner, CONCURRENT_IDLE, CONCURRENT_EMITTING)) {
```

Comment:
Use `acquire()` and `release` methods from above here too?

Reply:
This method is removed in https://github.com/apple/servicetalk/pull/1014, so lets discuss on that PR and for now I'll avoid touching in this PR.